src/mocks/auth/data.ts

```diff
@@ -0,0 +1,30 @@
+import { faker } from '@faker-js/faker';
+import { v4 as uuidv4 } from 'uuid';
+
+export type Person = {
+  entityId: string;
+  id: string;
+  name: string;
+  password: string;
+  joinDate: Date;
+};
+
+export const personList = [
+  {
+    entityId: 1,
+    id: 'journee@onboarding.com',
+    name: 'journee',
+    password: '123456',
+    joinDate: faker.date.past(),
+  },
+  ...Array.from({ length: 50 }, (_, i) => ({
```

Comment:
```suggestion
  ...Array.from({ length: 50 }, () => ({
```

여기 안 쓰는 변수들 지워줘도 좋을 것 같아요!